python/ray/train/BUILD

```diff
@@ -586,7 +594,7 @@ py_test(
 
 py_test(
     name = "test_trainer_restore",
-    size = "medium",
+    size = "large",
```

Comment:
why change this? does this PR have any impact on this test?

Reply:
the test was timing out for me, though you are correct, it should not be impacted by this. perhaps that was a fluke - I'll revert and see if it passes